src/main/java/core/basesyntax/supplier/FigureSupplier.java

```diff
@@ -0,0 +1,66 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.model.Circle;
+import core.basesyntax.model.Color;
+import core.basesyntax.model.Figure;
+import core.basesyntax.model.IsoscelesTrapezoid;
+import core.basesyntax.model.Rectangle;
+import core.basesyntax.model.RightTriangle;
+import core.basesyntax.model.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+
+    public Figure getRandomFigure() {
+        int number;
+        Random random = new Random();
+        ColorSupplier colorSupplier = new ColorSupplier();
+        number = random.nextInt(5);
+        switch (number) {
+            case 1: {
+                Circle circle = new Circle();
+                circle.setRadius(random.nextInt(100));
+                circle.setColor(colorSupplier.getRandomColor());
+                return circle;
+            }
```

Comment:
```suggestion
            case 0:
                return getRandomCircle();
```
lets add private methods to get each random figure - this would make your code easier to read